### ISSUE TITLE
Method to Assign Entries in One BJCP Category to Different Tables?

### ISSUE THREAD

**Issue by rfsilver** [geoffhumphrey/brewcompetitiononlineentry#1098]
Version: BCOE&M 2.1.17 Amateur Competition Edition

Installation URL: comp.gristhomebrew.com

Is your installation hosted on brewcompetition.com or brewcomp.com? No

Description of Issue: Is there a method to assign entries in one category to different tables? For example, I have two entries in 31A. Alternative Grain Beer. One is an American Brown Ale base that I would like to put on a table with other brown ales and the other is a Cream Ale base that I would like to put with similar pale beers.

Thank you!

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by geoffhumphrey** [comment]
Unfortunately, not at this time. Only entire sub-categories (styles) can be assigned to tables. 

I've put into the enhancement queue the ability to split certain styles, like 31A, among tables. Until then manual intervention is required.

**Comment by heneksj** [comment]
You can do it though.  I did it this year by creating a new style category, and putting that beer into the new category.  It is a little funky, but it works.  I did it after the labels were on the beer, and I included the original category in the title of the new phony category.  No one seemed to be too confused.  There are a lot of styles that this kind of thing comes up on.  In our case it was a 34C1 which I put into a phony 35A category so I could put it on a different table.

**Comment by AdrienM76** [comment]
I'll add a second vote for this being available across all styles and sub-styles. We just ran a comp this weekend where we wanted to break up the beers across different sets of judges and then do a mini-BOS to determine placement and advancement to BOS. But we couldn't figure out how to assign the styles to different tables. I see now the custom-category work around might have accomplished the task, but it would certainly be better if this just worked out of the box.

**Comment by ercousin** [comment]
@AdrienM76 it sounds like what you wanted was queued judging, where you can assign multiple pairs of judges to a single table/pullsheet.

**Comment by heneksj** [comment]
Yeah that’s the way I read @AdrienM76 ’s problem. If you need to just combine cats and do miniBOS, then you basically do queued judging and add enough judges so they can split the initial judging up. The system almost doesn’t have to know about it. 

The problem comes when you have a single subcategory that you would like to split off a beer or two onto another table, but not all the beers off to the other table. Like when a cat/sub can contain orphan beers that don’t really belong together. I suppose someone might also want to do something like separating the NEIPAs by strength—but our comp isn’t that big. 
The problem is kind of a corner case—but a better way would be nice.

**Comment by AdrienM76** [comment]
Our issue was two sub-categories we wanted to break up (21B - Specialty IPA, and 19C-American Brown) We wanted to spread them out across several tables so no one had to judge too many of the same thing. We managed with 21B because there are separate sub styles we could assign to tables individually. But 19C couldn't be split and we had 7 or 8 entries there we wanted to split in half but couldn't figure it out. We don't know if we'll even face this again or it might be in different categories. If queued judging lets us split sub-cats across tables then that is the route we'll take. But being able to assign any entry to any table would be nice. I'm not certain of the reason for forcing a sub-style on a single table and with that a single set of judges.

**Comment by heneksj** [comment]
Generally I like to keep similar beers together. Although that is hard sometimes. Sometimes you need to get creative.  I don’t think I would want to split 19c up. For me 7 or 8 is about enough for its own table.  But there are a lot of factors to consider and occasionally being able to split up a subcategory would be helpful. 

If you just randomly assign beers to tables, I don’t think the judges would like it, and how do you name the table and put on the ribbons or medals. What, would someone just get a ribbon for table 6? I don’t think that would be satisfying to the brewers either. I think I made a good table if I am able to give it a good name that makes sense.

**Comment by AdrienM76** [comment]
We give awards by main category, not table. I never suggested such a thing and I'm not looking for that. I simply wanted to offer that we had a use-case for splitting a sub-style. (those judges also had beers in 19A and 19B to evaluate) The ability (without a work-around) to assign some beers from a sub-style to a flight for a different set of judges would be helpful. I look forward to seeing it in v3.0 and if that makes it out in time for next January, we'll very likely need it.

**Comment by grmelacz** [comment]
Just my 2 cents. Here in Czechia, the competitions usually do not have separate tables for styles, but everyone does judge everything (sure, based on their preferred/not preferred styles). Just the styles are queued. For example, in first 2 hours, we do judge Czech Pale Lagers, then we have some Dunkels, after that American IPA and we finish with sours. But in one moment, one category might be judged by, say, 30 people sitting at 10 tables, each table having different beers.

So I would love to be able to define multiple tables with multiple styles on each table. The judges usually sit at one table for the whole judging, but there may be, of course, exceptions like having their beer in the competition (then a stand-in comes).
